### PR TITLE
Feature/activity form

### PR DESCRIPTION
**Pull request by tuj**
Added forms for activity

### CODE REVIEW

File: drupal/web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/checkbox.twig [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <label class="checkbox {{ checkbox.class }}">
   <div class="checkbox--label">{{ checkbox.label }}</div>
-  <input type="checkbox"/>
+  {% block content %}
+    <input type="checkbox"/>
```
Comment (magnify) [inline drupal/web/themes/custom/genlyd_aarhus/source/_patterns/00-elements/forms/checkbox.twig]: Remove end tags for void elements.

Void elements (e.g. <input> tags) should not have an end tag (/>) according to 
https://www.w3.org/TR/html51/syntax.html#void-elements

File: drupal/web/themes/custom/genlyd_aarhus/source/_patterns/03-templates/02-activity.twig [hunk 2]
```diff
@@ -1,41 +1,47 @@
...
-          <article class="grid--item">
-            {% include "@components/card-activity.twig" with { cardActivity: item } %}
-          </article>
-        {% endfor %}
+    {% embed "@layouts/container.twig" %}
+      {% block content %}
+        {% include "@components/activity-overview.twig" with { activityOverview: activity.activityOverview } %}
+
+        <div class="container has-large-spacing">
+          <div class="l-heading-section">
```
Comment (magnify) [inline drupal/web/themes/custom/genlyd_aarhus/source/_patterns/03-templates/02-activity.twig]: Replace html with embedded twig templates.

File: drupal/web/themes/custom/genlyd_aarhus/source/_patterns/03-templates/02-activity.twig [hunk 3]
```diff
@@ -1,41 +1,47 @@
...
+                <article class="grid--item">
+                  {% include "@components/card-activity.twig" with { cardActivity: item } %}
+                </article>
+              {% endfor %}
+            {% endblock %}
+          {% endembed %}
+
+        </div>
+
+        <div class="l-heading-section">
```
Comment (magnify) [inline drupal/web/themes/custom/genlyd_aarhus/source/_patterns/03-templates/02-activity.twig:27]: Replace with embeddded twig template.

File: drupal/web/themes/custom/genlyd_aarhus/source/_patterns/03-templates/02-activity.twig [hunk 4]
```diff
@@ -1,41 +1,47 @@
...
+        {% endembed %}
+
+        <div class="layout flex center">
+          {% include "@modules/call-to-action.twig" with { callToAction: activity.callToAction } %}
+        </div>
+
       {% endblock %}
     {% endembed %}
 
-    <div class="layout flex center">
```
Comment (magnify) [inline drupal/web/themes/custom/genlyd_aarhus/source/_patterns/03-templates/02-activity.twig]: Replace with embeddded twig template.

File: drupal/web/themes/custom/genlyd_aarhus/source/_patterns/03-templates/02-activity.twig [hunk 5]
```diff
@@ -1,41 +1,47 @@
...
+          {% block items %}
+            {% for item in activities|slice(4,7) %}
+              <article class="grid--item">
+                {% include "@components/card-activity.twig" with { cardActivity: item } %}
+              </article>
+            {% endfor %}
+          {% endblock %}
+        {% endembed %}
+
+        <div class="layout flex center">
```
Comment (magnify) [inline drupal/web/themes/custom/genlyd_aarhus/source/_patterns/03-templates/02-activity.twig:41]: Replace with embeddded twig template.

File: drupal/web/themes/custom/genlyd_aarhus/templates/content-edit/image-widget.html.twig [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+#}
+
+{#
+<div{{ attributes }}>
+  {{ data.preview }}
+  {{ data|without('preview') }}
+</div>
+#}
+
+<div class="image-upload">
```
Comment (magnify) [inline drupal/web/themes/custom/genlyd_aarhus/templates/content-edit/image-widget.html.twig]: This should be added as elements and modules to Pattern Lab and use embed.

File: drupal/web/themes/custom/genlyd_aarhus/templates/form/form--node-activity-form.html.twig [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+                ] } %}
+                {% include '@modules/progress-bar.twig' %}
+              {% endblock %}
+            {% endembed %}
+      #}
+
+
+{% set form = element %}
+
+<form{{ attributes }}>
```
Comment (magnify) [inline drupal/web/themes/custom/genlyd_aarhus/templates/form/form--node-activity-form.html.twig:35]: Add as component in Pattern Lab and embed so we avoid maintaining multiple components.

File: drupal/web/themes/custom/genlyd_aarhus/templates/form/form--node-activity-form.html.twig [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+            {% block items %}
+              {% embed "@layouts/grid-item.twig" %}
+                {% block content %}
+                  {{ form.field_image }}
+                {% endblock %}
+              {% endembed %}
+
+              {% embed "@layouts/grid-item.twig" %}
+                {% block content %}
+                  {{ "Her kan du give din aktivitet et billede. Vælg upload billede og tilføj et billede fra din computer." | trans }}
```
Comment (magnify) [inline drupal/web/themes/custom/genlyd_aarhus/templates/form/form--node-activity-form.html.twig:137]: Original text should be in english.

**Review by magnify — CHANGES_REQUESTED** [review]

**Comment by tuj** [comment]
Changes made to branch

**Pull request merged**